internals/updates/nomad.go

```diff
@@ -0,0 +1,62 @@
+package updates
+
+import (
+	"context"
+
+	nomadapi "github.com/hashicorp/nomad/api"
+	"github.com/pkg/errors"
+	"github.com/rs/zerolog/log"
+)
+
+type NomadUpdaterCleaner struct {
+	updateID int
+}
+
+func NewNomadUpdaterCleaner(updateID int) *NomadUpdaterCleaner {
+	return &NomadUpdaterCleaner{
+		updateID: updateID,
+	}
+}
+
+func (nu *NomadUpdaterCleaner) Clean(ctx context.Context) error {
+	// Create a Nomad API client configuration
+	client, err := nomadapi.NewClient(nomadapi.DefaultConfig())
+	if err != nil {
+		return errors.Wrap(err, "failed to init Nomad api client")
+	}
+
+	// Remove the job
+	jobID := "portainer-updater"
+
+	job, _, err := client.Jobs().Info(jobID, nil)
+	if err != nil {
+		log.Debug().
+			Err(err).
+			Msg("failed to find nomad job portainer-updater")
+		return errors.Wrap(err, "failed to find nomad job portainer-updater")
```

Comment:
if we didn't find the job, it's ok, no? I think in this case just `return nil`

Reply:
Right. I think I didn't change back after debugging.